index.html

```diff
@@ -25,8 +25,18 @@
       },
       {
         name: "Yuri Shkuro",
-        company: "Facebook",
+        company: "Meta",
```

Comment:
@yurishkuro FYI updating your employer's name if you are okay with that.

Reply:
+1